lib/dockerfile/instruction-parser.ts

```diff
@@ -64,6 +68,17 @@ function getPackagesFromRunInstructions(runInstructions: string[]) {
   }, {});
 }
 
+function removeRunDefFromInstruction(instruction: string) {
+  let cleanedInstruction = instruction;
+  const runDefs = ["RUN ", "/bin/sh ", "RUN /bin/sh"];
```

Comment:
what about the `-c` ? of `/bin/sh -c`

Reply:
That's the answer to the other question - https://github.com/snyk/snyk-docker-plugin/pull/312#discussion_r548010339 :)